docs/core/testing/unit-testing-fsharp-with-dotnet-test.md

```diff
@@ -64,6 +64,8 @@ The test project requires other packages to create and run unit tests. `dotnet n
 dotnet add reference ../MathService/MathService.fsproj
 ```
 
+[!INCLUDE [testing an ASP.NET Core project from .NET Core](~/includes/selective-unit-tests.md)]
```

Comment:
Same comment applies to all files.
Although ~ is more practical, when I can, I use the relative ones to make it easier for people to find the files when coming to edit them on GitHub from docs. And the new file is called core-testing-note-aspnet.md, right?

```suggestion
[!INCLUDE [testing an ASP.NET Core project from .NET Core](../../../includes/core-testing-note-aspnet.md)]
```

Reply:
Weird, i dont know why that name got in there.

Reply:
strange, it's fine on my machine. there must have been a desync with github. I'll reset my local branch.